source/release-notes/3-1-14.rst

```diff
@@ -72,6 +69,9 @@ For a full list of tickets, please visit: `GitHub 3.1.14 Milestone <https://gith
 
     For Docker/Kuber deployments, see :ref:`upgrading-dockers`
 
+    Java has decided to disable support for TLS versions minor to 1.2 in the latest Java version 1.8.0_292.
```

Comment:
Please change to:
"Starting with JVM version 1.8.0_292, support for TLS v1.1 and earlier is disabled by default."